### PR TITLE
Merge changes from beta 19.7.0.1 + Release Notes

### PR DESCRIPTION
**Pull request by mokagio**
Includes:

- [[Site Name] Adjusts design picker buttons for treatment variation](https://github.com/wordpress-mobile/WordPress-iOS/pull/18397)  (they now say "Create Site" instead of "Choose") by @antonis and @twstokes 

Nothing to test, if CI Shows green, we're good.

## Regression Notes

1. Potential unintended areas of impact – N.A.
2. What I did to test those areas of impact (or what existing automated tests I relied on) – N.A.
3. What automated tests I added (or what prevented me from doing so) – N.A.

---

- [x] I have completed the Regression Notes.
- [x] I have considered adding unit tests for my changes. **N.A.**
- [x] I have considered adding accessibility improvements for my changes. **N.A.**
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary. **N.A.**

### CODE REVIEW

**Review by mokagio — COMMENTED** [review]
- [x] Updates to the localization files (`.strings` and Fastlane metadata) automatically pulled for the new translations that have already been approved in GlotPress
- [x] Version update in `.xcconfig`
- [x] Diffs from the PRs that made it into this beta

File: RELEASE-NOTES.txt
Comment (AliSoftware) [inline RELEASE-NOTES.txt:9]: ❓ Is that _change_ expected (especially since that does not seem like just a wording update of a single item, but instead removal of an item related to one PR #16106 while adding a different item related to a different PR #18127)?
(cc @twstokes as author of the commit making that change)

And also, do we need to inform our freelance writer (Grace) about that change and new entry, so that she update her Editorialized Release Notes copy (included in this PR) accordingly?

File: WordPress/Classes/ViewRelated/Site Creation/Design Selection/Preview/SiteDesignPreviewViewController.swift
Comment (AliSoftware) [inline WordPress/Classes/ViewRelated/Site Creation/Design Selection/Preview/SiteDesignPreviewViewController.swift:34]: ⚠️ Note that since those 2 new strings were not present at the time you did the code freeze and got added in `release/*` later, those new strings sadly won't be sent for translation and won't be translated in time for 19.7 😞

File: WordPress/Jetpack/Resources/AppStoreStrings.po
Comment (AliSoftware) [inline WordPress/Jetpack/Resources/AppStoreStrings.po:53]: 👍  I too feel this wording is safer than the more spicy 🌶️  one Grace propose 😅  (While it was funny, the alternative spicier copy might have been a lost reference that not everybody would have got, and could also have been negatively interpreted)

File: WordPress/WordPress.xcodeproj/project.pbxproj
Comment (AliSoftware) [inline WordPress/WordPress.xcodeproj/project.pbxproj:2172]: Woohoo, more tests! 🎉 ❤️

File: fastlane/metadata/tr/subtitle.txt
Comment (AliSoftware) [inline fastlane/metadata/tr/subtitle.txt:1]: Still <30, all good 👍 😌

**Review by AliSoftware — COMMENTED** [review]
#### Checklist for New Beta

 - [X] Changes from bugfix PRs which landed in the release branch
 - [X] Version bump in `.xcconfig` files
 - [X] Latest app and metadata translations pulled from Glotpress

#### Checklist for Release Notes

 - [X] Editorialized Release Notes updated
 - ⚠️  entries from `RELEASE-NOTES.txt` changed in the `release/*` branch 🤔  quite suspicious, and not sure if this was intentional or not, nor if it would require us to revisit the Editorizalized Release Notes accordingly to take that change into account…
 - ⚠️ New strings were introduced post-code-freeze in the `release/*` branch (and thus post-strings-freeze too)
    - Which means those new strings won't be included in the translation pipeline (nor sent to our Translation Vendor nor imported in GlotPress), and thus not translated in time for 19.7.
    - @mokagio As a mitigation for this, you might want to consider updating the frozen strings (i.e. run the `generate_strings_file_for_glotpress` again on `release/19.7` branch) [similar to what I decided to do for WPAndroid](https://github.com/wordpress-mobile/WordPress-Android/pull/16376#discussion_r855058494), so that the new strings from the `release/19.7` branch will be imported into GlotPress. While the WeeklyKit has already been sent and it would be too late for those to get to our translation vendor, this would still give a chance to the polyglots community to help translate those new strings in time for 19.7 submission.
    - (cc @antonis FYI, as author of those new strings)

File: RELEASE-NOTES.txt
Comment (twstokes) [reply RELEASE-NOTES.txt:9]: 👋 @AliSoftware - [this PR contains the backstory of this change](https://github.com/wordpress-mobile/WordPress-iOS/pull/18401).

Summary:
- The work started out as a fix for https://github.com/wordpress-mobile/WordPress-iOS/issues/16106 by an external contributor
- During the review process we made them aware that the fix had broader scope than just reader
- At the tail end (a little too late) I updated their contributed release note and revised their PR title to better reflect the changes. It was [pointed to the issue](https://github.com/wordpress-mobile/WordPress-iOS/issues/16106) rather than the PR, so I fixed that as well. I then created https://github.com/wordpress-mobile/WordPress-iOS/pull/18401 to hopefully prevent any surprises on the release side.

Should I have included the Owl Team on that PR to loop everyone in? Any tips welcome! 🙇

File: WordPress/Classes/ViewRelated/Site Creation/Design Selection/Preview/SiteDesignPreviewViewController.swift
Comment (twstokes) [reply WordPress/Classes/ViewRelated/Site Creation/Design Selection/Preview/SiteDesignPreviewViewController.swift:34]: 👋 @AliSoftware thanks for all your information recently about translations. Could I get one bit clarified?

If I understand correctly, "Create Site" and "Choose" are keys* for the translated strings. These keys existed prior to this change in WPiOS and had the same meaning [[one](https://github.com/wordpress-mobile/WordPress-iOS/blob/7ea0fb21958cbd1516e236b99b1b28322b776f67/WordPress/Resources/en.lproj/Localizable.strings#L2121)] [[two](https://github.com/wordpress-mobile/WordPress-iOS/blob/7ea0fb21958cbd1516e236b99b1b28322b776f67/WordPress/Resources/en.lproj/Localizable.strings#L1462)].

Does this still mean the translations won't work for this view? I ran it locally and set it to a random language (German) and saw it was translated. Thanks!

*I realize we may be getting away from the English string being the key soon.

File: RELEASE-NOTES.txt
Comment (AliSoftware) [reply RELEASE-NOTES.txt:9]: Thanks for the helpful extra context @twstokes !

I think it's always a good idea to ping your Release Manager especially when you change either release notes (like here) or strings (like for the case of the other review comment I left) in a PR that lands in `release/*` branch after the code-freeze.

Both release notes and strings are frozen during code freeze and we then use those frozen versions to kick off the rest of the process (ask our freelance writer to write nice notes from this frozen draft, send frozen strings to our translation vendor, …), which is why any change to those after code freeze is important to signal to your RM so they can adjust for those last-minute tweaks appropriately 🙂.

---

That being said, given the additional context you shared here, I think the wording that ended up being used in the Editorialized version of the Release Notes [here below](https://github.com/wordpress-mobile/WordPress-iOS/pull/18411/files#diff-a0ae2ed3128fff2f7c059edd7fe0d51f6a7f1ec9ecea83b6d46f6b8d16a7d3bcR7) is probably OK and matches what ultimately landed?

File: WordPress/Classes/ViewRelated/Site Creation/Design Selection/Preview/SiteDesignPreviewViewController.swift
Comment (AliSoftware) [reply WordPress/Classes/ViewRelated/Site Creation/Design Selection/Preview/SiteDesignPreviewViewController.swift:34]: Ah! I might have missed something when looking at the diff then, as I was convinced that those were newly introduced strings — as opposed to already existing ones that we just moved around or reused in a different context.

If they were already part of the strings that got extracted in the generated `Localizable.strings` back when Gio did the code freeze (maybe because they were already used / present in another part of the code) then we should be all good for those then!

Comment (twstokes) [reply WordPress/Classes/ViewRelated/Site Creation/Design Selection/Preview/SiteDesignPreviewViewController.swift:34]: Cool! 

"Create Site" is a "new" string to that view, but it was used (in the same context) in WPiOS prior to this change, so my hope is that the translation for that key is just reused.

"Choose" already existed on this view, it was only moved. 👍

File: RELEASE-NOTES.txt
Comment (twstokes) [reply RELEASE-NOTES.txt:9]: Thanks @AliSoftware, that makes sense!

> I think the wording that ended up being used in the Editorialized version of the Release Notes [here below](https://github.com/wordpress-mobile/WordPress-iOS/pull/18411/files#diff-a0ae2ed3128fff2f7c059edd7fe0d51f6a7f1ec9ecea83b6d46f6b8d16a7d3bcR7) is probably OK and matches what ultimately landed?

I agree!

**Review by AliSoftware — APPROVED** [review]
Approving now that my two previous ⚠️ / questions were answered by @twstokes in the associated comments 👍🙇‍♂️

**Pull request merged**
